src/test/java/org/killbill/billing/plugin/adyen/api/mapping/KlarnaPaymentMappingServiceTest.java

```diff
@@ -130,4 +133,44 @@ public void testAuthResponseData() throws Exception {
         assertEquals(paymentInfo.getDetailsData().get("key1"), "ResponseData1");
         assertEquals(paymentInfo.getDetailsData().get("key2"), "ResponseData2");
     }
+
+    @Test(groups = "fast")
```

Comment:
Can you break this test to small tests with specific scenarios they are testing for.

Reply:
This has been updated in latest PR#6